src/store/userreducerActions.js

```diff
@@ -0,0 +1,2 @@
+export const changeUser = 'CHANGE_USER'
+export const logoutUser = 'LOGOUT_USER'
```

Comment:
always put a new line at the end of the file  -> which goes back to using a linter again